modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -644,7 +644,7 @@ Whenever users log in, their display name, email, quota, avatar and search attri
 If you want to keep the metadata up to date you can set up a cron job, using xref:configuration/server/occ_command.adoc#syncing-user-accounts[the occ user:sync command].
 Versions of ownCloud before 10.0 imported all users when the users page was loaded, but this is no longer the case.
 
-TIP: We recommend xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[creating a Cron job], to automate regularly syncing LDAP users with your ownCloud database.
+TIP: We recommend xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[creating a Cron job], to automate regularly syncing LDAP users with your ownCloud database. You can use the xref:developer_manual:core/apis/ocs/user-sync-api.adoc[the OCS User Sync API] to simplify the process.
```

Comment:
- You have double "the"
- I don't think that the user sync API replaces or simplifies the cron job. It rather provides a way to trigger user sync from the outside which helps as the external user provisioning system can tell ownCloud when there are changes (push vs. pull). To put it differently: If you have many users you do not have to sync all of them (pull) to get one new user. Instead the external user backend "reports" that one new user via the API (push).
- Usually, you would still use the cron job to sync users.

Reply:
I updated the section based on your feedback, @pmaier1.